tests/test_graphical_units/test_axes.py

```diff
@@ -20,6 +20,17 @@ def construct(self):
         self.add(graph, labels)
 
 
+class PlotFunctionWithDiscontinuitiesTest(ThreeDScene):
```

Comment:
ThreeDScene seems inappropriate, maybe the tests are failing because of that? (Didn't check the logs yet.)

```suggestion
class PlotFunctionWithDiscontinuitiesTest(Scene):
```

Reply:
Typo, sorry about that